### PR TITLE
Add .circleci/config.yml

### PR DESCRIPTION
**Pull request by mjgiarlo**
## Why was this change made?

To add CI

## How was this change tested?

... CI ?

## Which documentation and/or configurations were updated?

CI config

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    executor: ruby/default
+    steps:
+      - checkout
+      - run:
+          name: Which bundler?
+          command: bundle -v
+      - ruby/install-deps
+      - run:
+          name: Lint using Rubocop
+          command: bundle exec rubocop
```
Comment (justinlittman) [inline .circleci/config.yml:25]: Use https://circleci.com/developer/orbs/orb/circleci/ruby#commands-rubocop-check?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Invoke jobs via workflows
+# See: https://circleci.com/docs/2.0/configuration-reference/#workflows
+workflows:
+  sample: # This is the name of the workflow, feel free to change it to better match your workflow.
```
Comment (justinlittman) [inline .circleci/config.yml:30]: Change? Cruft?

File: .circleci/config.yml (hunk 1 above)
Comment (mjgiarlo) [reply .circleci/config.yml:25]: @justinlittman I'll take a look, thanks!

File: .circleci/config.yml (hunk 2 above)
Comment (mjgiarlo) [reply .circleci/config.yml:30]: circleci-generated cruft. will cleanup.

**Pull request merged**